src/edu/wright/cs/raiderplanner/model/Exam.java

```diff
@@ -45,26 +45,51 @@ protected void replace(VersionControlEntity receivedVce) {
 		super.replace(receivedVce);
 	}
 
-	// public methods
-
-	// getters
+	/**
+	 * This Getter method gets the Resit value.
+	 * @return
+	 */
 	public Exam getResit() {
 		return resit;
 	}
 
+	/**
+	 * Get's the Exam event time slot.
```

Comment:
This should have been changed when you fixes the other occurrence of `Get's` in this file.